packages/core/src/test/dynamoDb/utils/dynamodb.test.ts

```diff
@@ -0,0 +1,161 @@
+                '',
+                dynamoDbClient
+            )
+
+            assert.ok(scanTableStub.called)
+            assert.deepEqual(actualResult, getExpectedResult())
+        })
+
+        it('Empty result should be handled', async () => {
+            const expectedScanResult = {
+                Items: [],
+            } as unknown as ScanOutput
+            sinon.stub(dynamoDbClient, 'scanTable').resolves(generateRequest(expectedScanResult))
+
+            const actualResult = await dynamoDbUtils.getTableContent(
+                { TableName: 'Users', Limit: 5 },
+                '',
+                dynamoDbClient
+            )
+
+            const expectedResult: dynamoDbUtils.TableData = {
+                lastEvaluatedKey: undefined,
+                tableContent: [],
+                tableHeader: [],
+            }
+            assert.deepEqual(actualResult, expectedResult)
+        })
+    })
+
+    describe('copyDynamoDbArn', function () {
+        it('should copy ARN', async () => {
+            const expectedResult = { TableName: 'Table1', TableArn: 'tablearn' } as unknown as TableDescription
+
+            const dynamoDbTableNode = sinon.stub() as unknown as DynamoDbTableNode
+
+            const getTableInformationStub = sinon.stub(dynamoDbClient, 'getTableInformation').resolves(expectedResult)
+
+            await dynamoDbUtils.copyDynamoDbArn(dynamoDbTableNode, dynamoDbClient)
+            assert.ok(getTableInformationStub.called)
+        })
+
+        it('shoudl not copy to clipboard', async () => {
+            const expectedResult = { TableName: 'Table1' } as unknown as TableDescription
+
+            const dynamoDbTableNode = sinon.stub() as unknown as DynamoDbTableNode
+            const copyToClipboardStub = sinon.stub(utilities, 'copyToClipboard').resolves()
+
+            const getTableInformationStub = sinon.stub(dynamoDbClient, 'getTableInformation').resolves(expectedResult)
+
+            await dynamoDbUtils.copyDynamoDbArn(dynamoDbTableNode, dynamoDbClient)
+            assert.ok(getTableInformationStub.called)
+            assert.equal(copyToClipboardStub.called, false)
+        })
+    })
+
+    describe('queryTableContent', function () {
+        it('should query the table', async () => {
+            const expectedQueryResult = {
+                Items: [{ age: { N: '25' }, ID: { S: '2' }, name: { S: 'Jane' } }],
+            } as unknown as ScanOutput
+
+            sinon.stub(dynamoDbClient, 'getTableInformation').resolves(getTableDescriptionResponse())
+            sinon.stub(dynamoDbClient, 'queryTable').resolves(generateRequest(expectedQueryResult))
+            const actualResult = await dynamoDbUtils.queryTableContent(
+                { partitionKey: 'library', sortKey: 'as' },
+                'regionCode',
+                'ableName',
+                dynamoDbClient
+            )
+
+            assert.deepEqual(actualResult, getExpectedResult())
```

Comment:
Look to always use the `strict` version as it enforces the type as well, otherwise 1 and "1" can be equal: https://stackoverflow.com/questions/16745855/difference-between-equal-deep-equal-and-strict-equal

Reply:
Understood I will update it